src/Content/Backend/Solution/Monaco.Template.Backend.Api/Endpoints/Extensions/EndpointsExtensions.cs

```diff
@@ -17,7 +17,12 @@ public static IEndpointRouteBuilder RegisterEndpoints(this IEndpointRouteBuilder
 
 		return builder.AddCompanies(versionSet)
 					  .AddCountries(versionSet)
+#if (filesSupport)
 					  .AddFiles(versionSet)
 					  .AddProducts(versionSet);
+#else
+					  .AddCountries(versionSet);
```

Comment:
I think this will include `.AddCountries(versionSet)` a second time when the `filesSupport` flag is `false`?

Reply:
You're right! I missed the correct position of the #if... Should be fixed now.